### PR TITLE
Fixing log levels analyzer

### PR DESCRIPTION
**Pull request by manumafe98**
See [#150](https://github.com/exercism/java-analyzer/issues/150)

TODO:
- [x] Update substring comment on website-copy
- [x] Merge https://github.com/exercism/website-copy/pull/2333

### CODE REVIEW

File: src/main/java/analyzer/exercises/loglevels/UseSubstringMethod.java [hunk 1]
```diff
@@ -2,16 +2,30 @@
 
 import analyzer.Comment;
 
+import java.util.Map;
+
 /**
  * @see <a href="https://github.com/exercism/website-copy/blob/main/analyzer-comments/java/log-levels/use_substring_method.md">Markdown Template</a>
  */
 class UseSubstringMethod extends Comment {
+    private final String onMethod;
```
Comment (sanderploegsma) [inline src/main/java/analyzer/exercises/loglevels/UseSubstringMethod.java:11]: _Very_ small improvement: you don't use substring _on_ a method, but _in_ a method. So I suggest renaming the parameter to `inMethod`.

**Review by sanderploegsma — APPROVED** [review]
One minor detail, the rest LGTM!

File: src/main/java/analyzer/exercises/loglevels/UseSubstringMethod.java (hunk 1 above)
Comment (manumafe98) [reply src/main/java/analyzer/exercises/loglevels/UseSubstringMethod.java]: Done!

**Comment by sanderploegsma** [comment]
Let's solve that in a separate PR. For now these changes at least clear up which method the feedback is addressing, which should prevent some confusion for students.

File: src/main/java/analyzer/exercises/loglevels/UseSubstringMethod.java [hunk 2]
```diff
@@ -2,16 +2,29 @@
...
 class UseSubstringMethod extends Comment {
+    private final String inMethod;
+
+    public UseSubstringMethod(String inMethod) {
+        this.inMethod = inMethod;
+    }
 
     @Override
     public String getKey() {
         return "java.log-levels.use_substring_method";
```
Comment (sanderploegsma) [inline src/main/java/analyzer/exercises/loglevels/UseSubstringMethod.java:19]: This should change to `java.log-levels.use_substring_method_v2`, see https://github.com/exercism/website-copy/pull/2333.

Also update the Javadoc comment at the top to point to the new file

**Review by sanderploegsma — CHANGES_REQUESTED** [review]

File: src/main/java/analyzer/exercises/loglevels/UseSubstringMethod.java (hunk 2 above)
Comment (manumafe98) [reply src/main/java/analyzer/exercises/loglevels/UseSubstringMethod.java]: Done!

**Review by sanderploegsma — APPROVED** [review]

**Comment by manumafe98** [comment]
> Let's solve that in a separate PR. For now these changes at least clear up which method the feedback is addressing, which should prevent some confusion for students.

Sure, where we discuss it, in a new issue ?

**Comment by sanderploegsma** [comment]
No need for a new issue, we can just use the existing issue to continue the conversation. Otherwise it would be harder to track.

I updated the PR description so that it no longer auto-closes the issue, so it can remain open until we're satisfied.

**Comment by manumafe98** [comment]
> No need for a new issue, we can just use the existing issue to continue the conversation. Otherwise it would be harder to track.
> 
> I updated the PR description so that it no longer auto-closes the issue, so it can remain open until we're satisfied.

Perfect!

**Comment by manumafe98** [comment]
@ErikSchierboom Hey Erik! Can you please give us a hand with the review of this pr? Thanks!

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**